docs/src/content/docs/fr/guides/customization.mdx

```diff
@@ -139,13 +139,13 @@ tableOfContents:
   </TabItem>
   <TabItem label="Configuration globale">
 
-```js
+```js {7}
 // astro.config.mjs
 
 defineConfig({
 	integrations: [
 		starlight({
-			title: '',
+			title: 'Site avec une configuration de table des matières personnalisée',
```

Comment:
Took the liberty to use "Site" here instead of "Documentation" for 2 reasons:

- Match the example right below
- Prettier with the new longer length will break this in 2 lines

Let me know if "Documentation" is preferred here.

Reply:
This works! Happy to have these tweaked contextually.